bridge_adaptivity/module/views.py

```diff
@@ -406,3 +406,17 @@ def callback_sequence_item_grade(request):
         policy.send_grade()
 
     return HttpResponse(xml, content_type="application/xml")
+
+
+def sync_collection(request, pk):
+    """
+    Synchronize collection immediately.
+    """
+    back_url = request.GET.get('back_url')
+    collection = Collection.objects.get(pk=pk)
+    collection.save()
+    log.debug("Immediate sync task is created, time: {}".format(collection.updated_at))
+    tasks.sync_collection_engines.delay(
+        collection_id=pk, created_at=collection.updated_at
+    )
+    return redirect(reverse('module:collection-detail', kwargs={'pk': pk}) + '?back_url={}'.format(back_url))
```

Comment:
```
In [6]: 'back_url={}'.format(None)
Out[6]: 'back_url=None'
```
if back_url is None it will return not correct url. Am I miss something?

Reply:
if back url is None back button won't be added

Reply:
After checking back_url is always added in the template.